solutions/USACO dec20-cowntagion.mdx

```diff
@@ -2,14 +2,76 @@
 id: usaco-1062
 source: USACO Silver 2020 December
 title: Cowntagion
-author: Kevin Sheng
+author: Kevin Sheng (Java, Python), Tanish Tyagi (C++)
 ---
 
 [Official Editorial](http://www.usaco.org/current/data/sol_prob1_silver_dec20.html)
 
+We can create a graph based off the roads between the farms which is given as input.
+The fastest way for COWVID-19 to spread to all the farms is for a farm to have superspreader events until 1 cow can be sent to every adjacent farm. This would start at the root node, and repeat until COWVID-19 has spread to all the farms. 
```

Comment:
Not very thorough. If you don't already know the solution, this doesn't make much sense.

Reply:
add smth like "see the official analysis for proof".

Reply:
It's not whether it's rigorous -- It's hard to understand what's going on.

"a farm to have superspreader events until 1 cow can be sent to every adjacent farm" which farm? what about the non-root farms?

It's just not very intuitive to read.